react/react1/week1/todo-app/src/components/TableLIst.jsx

```diff
@@ -0,0 +1,17 @@
+import TaskRow from "./TaskRow";
+
+const TableList = ({todos}) => {
+    return (
+        <>
+            {todos.map((todo) => 
+                <TaskRow 
+                description={todo.description} 
+                deadline={todo.deadline} 
+                key={todo.id} 
```

Comment:
Nice job. Still, if you haven't already, please read the docs for why we are doing it
https://react.dev/learn/rendering-lists

Reply:
@YurijPaharev  Thank you a lot for the review! 🙌